stats/s-bandwidth

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+LKP_SRC = ENV['LKP_SRC'] || File.dirname(File.dirname(File.realpath($PROGRAM_NAME)))
+
+require "#{LKP_SRC}/lib/statistics"
+
+workload = 0
+data = false
+pattern = /^\S*\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+/
+while (line = STDIN.gets)
+  if [data]
```

Comment:
thanks, would "if data" is enough here without the Array form?

Reply:
done